src/server.rs

```diff
@@ -254,19 +254,25 @@ struct Cli {
 fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
     let data_dir = util::get_default_data_dir();
 
-    // Initialize the logger
-    let log_writer = util::init_logger(&data_dir)?;
-
-    // Initialize the logger and log the log file path
-    log::info!("Logging initialized. Log file located at: {:?}", data_dir.join("debug.log"));
-
+    // Read the configuration file
     let contents = fs::read_to_string("../../example-config.toml")
         .expect("Something went wrong reading the file");
 
     let config: Config = toml::from_str(&contents)
```

Comment:
If there is no config file, what do you think to have a `Config::default()` called with the same value than in `example-config.toml` ?

Reply:
It makes sense. Im testing this and seems to be working. Ive pushed the code. Let me know how it works? Seems standard practice and good to implement

Reply:
Yes tested and the code works.